core/src/main/kotlin/org/eln2/space/Location.kt

```diff
@@ -210,10 +293,15 @@ data class BlockPos(override val vec3i: Vec3i) : Locator {
 
 	override fun neighbors(): List<Locator> = CONNECTIVITY_DELTAS.map { translate(it) }
 
+	/**
+	 * Offsets a vector based on the position of this node.
+	 */
 	fun translate(v: Vec3i) = BlockPos(vec3i + v)
 }
 
-/* Support for surface-mounted nodes, up to six per block. */
+/**
+ *  Positionally support for surface-mounted nodes, up to six per block.
```

Comment:
"Locator support", perhaps, and you might also include "in three dimensions" again to be clear.